ecocodejava.properties

```diff
@@ -1,7 +1,9 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Java language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/ecoCode
-publicVersions=1.2.1
+publicVersions=1.3.0
+archivedVersions=1.2.1
```

Comment:
archived versions can't be compatible with `LATEST`. And only one version per plugin can be compatible with `LATEST`

Reply:
so sorry. I forgot this point.
I've fixed-up